pymagicc/MAGICC6/run/TCRECS_CO2_CONC.IN

```diff
@@ -1,3415 +0,0 @@
-TCR Diagnosis input file
-Written by: diagnose-tcr notebook in pymagicc package
-
-&THISFILE_SPECIFICATIONS
-    THISFILE_DATACOLUMNS = 1
```

Comment:
Can the TCRECS_*_CONC.IN from magicc7 also be deleted?

Reply:
yep